package.json

```diff
@@ -29,7 +30,7 @@
     "next": "13.1.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "stytch": "^6.0.0-next.0",
+    "stytch": "^6.0.7",
```

Comment:
Is it okay to use this version or did you include the `next` for any given reason?

Reply:
`next` was a prerelease tag - we're fine to use the latest version when OIDC is published